docs/Reference/API-Objects.md

```diff
@@ -83,7 +83,8 @@ Returned by [priv_getPrivateTransaction](API-Methods.md#priv_getprivatetransacti
 | **r**                | Data, 32&nbsp;bytes               | ECDSA signature r                                                               |
 | **s**                | Data, 32&nbsp;bytes               | ECDSA signature s                                                               |
 | **privateFrom**      | Data, 32&nbsp;bytes               | [Orion](https://docs.orion.pegasys.tech/en/stable/) public key of sender                                                      |
-| **privateFor**       | Array of Data, 32&nbsp;bytes each | [Orion](https://docs.orion.pegasys.tech/en/stable/) public keys of recipients                                                 |
+| **privateFor**       | Array of Data, 32&nbsp;bytes each | [Orion](https://docs.orion.pegasys.tech/en/stable/) public keys of recipients. Not returned if `privacyGroupId` is used to [send the transaction](../Concepts/Privacy/Privacy-Groups.md#Privacy-Types).      |
```

Comment:
When I published this locally the link on [send the transaction] linked to the Privacy Groups page but not the Privacy Types section.  The section name needs to be lower case in link.  Link checker doesn't pick up whether the links to specific sections are correct so it's a good idea to click on these when you publish locally to check that they're correct.